feat: update and mongo cheat (#2)
LucasSGomide/MongoDB: README.md

  },
  { upsert: true }
);
```

### $ALL

Selects the documents where the value of a field is an array that contains all the specified elements.

``` javascript
db.inventory.find(
  { tags: { $all: ["red", "blank"] } }
);
```

### $ELEMMATCH

> Select documents that contain an array field with at least one element that matches all the specified.

Reviewer: ```suggestion
Select documents that contain an array field with at least one element that matches all the specified filters.
```

``` javascript
db.scores.find(
  { results: { $elemMatch: { $gte: 80, $lt: 85 } } }
);
```

### $SIZE

> Select matches any array with the number of elements specified by the argument.

Reviewer: ```suggestion
Select documents that match any array with the number of elements specified by the argument.
```

``` javascript
db.products.find(
  { tags: { $size: 2 } }
);
```

---

## Evaluation Operators

### $WHERE

Use to pass either a string containing a JavaScript expression or a full JavaScript function.

Starting in MongoDB 3.6, the **$expr** operator allows the use of aggregation expressions.

If you must create custom expressions, $function is preferred over **$where**.

### $EXPR

> Allows the use of [aggregation](#aggregation) expressions.

Reviewer: ```suggestion
Allows the use of [aggregation](#aggregation) expressions. It compares fields from **the same** document.
```
